includes/gateways/stripe/includes/admin/admin-helpers.php

```diff
@@ -204,3 +179,22 @@ function give_stripe_connect_delete_options() {
 	give_delete_option( 'live_publishable_key' );
 	give_delete_option( 'test_publishable_key' );
 }
+
+/**
+ * This helper function is used to convert slug into names of Stripe connection.
+ *
+ * @param string $slug Connection Type Slug.
+ *
+ * @since 2.7.0
+ *
+ * @return string
+ */
+function give_stripe_connection_type_names( $slug = 'connect' ) {
```

Comment:
@mehul0810 Are we using this function get label for connecting type if yes then function name can be different then existing.
```suggestion
function give_stripe_connection_type_name( $slug = 'connect' ) {
```